README.md

```diff
@@ -2,6 +2,20 @@ Own Up Grown Up is a scheduling and reminder application that bridges the gap be
 
 Own Up Grown Up was created by TEAM JACK (Julianne Peters, Andrew Garrett, Chris Stubbs, and Kristen McCloud) and includes professional design and security.  Using our application and the Twilio API, you will always get your reminder and always be able to change your personal preferences.
 
-Don't forget to remember, or just use Own Up Grown Up. 
+Don't forget to remember, or just use Own Up Grown Up.
 # back2Point0
 # front2Point0
+
+
+TODO:
+  Same logic used in myTaskCard and taskCard to choose icons; we need to move up the state tree to DRY
+  Add id to <i> tags so that if no internet connection, still shows something
+  Move stylesheets to corresponding page folders
+  App.css has a .testing class - what is it used for?
+  bodyBox in taskCard and myTaskCard is assigned but never defined in any css file
+  Media query for navbar for transition from full to smaller window
```

Comment:
There is already a transition for the side menu. Do you want the top navbar to just be completely gone?

Reply:
So on my computer, there is a moment when you adjust screen sizes that the navbar completely vanishes. I'm not sure if that will ever really be an issue, but it seems like it might be pretty simple to solve? IDK though; you're the style guru ;)

Reply:
Got it! It's a media query issue. I'll look at it! :-). For sure a easy fix. Probably just have a px off in one of them....